db/migrate/20220119201757_add_column_default_to_rd_active.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnDefaultToRdActive < ActiveRecord::Migration
+  def change
+    change_column_null :recurring_donations, :active, true, false
```

Comment:
What would happen if we did `change_column_null :recurring_donations, :active, false, false` instead? Would it be not nullable?

Reply:
Wait, I think this is wrong! It's so confusing

```ruby
change_column_null :table_name, :column_name, nulls_are_allowed, what_to_convert_nulls_to
```